quetz/config.toml

```diff
@@ -0,0 +1,11 @@
+[github]
+# Register the app here: https://github.com/settings/applications/new
+client_id = "{}"
+client_secret = "{}"
+
+[sqlalchemy]
+database_url = "{}"
+
+[session]
+secret = "{}"
+https_only = false
```

Comment:
The default value of https_only was `true`.

Reply:
not in dev_config.toml, but I will change that!

Reply:
Aye, since for a dev environment you're unlikely to bother with SSL certs etc.